packages/unpublished-eslint-rules/index.js

```diff
@@ -0,0 +1,288 @@
+const TypeParamNamingSchema = [['T'], ['K', 'V'], ['T', 'K', 'V'], ['T', 'K', 'V', 'R']];
```

Comment:
Is the structure of this code copied and adapter from somewhere or is this a 100% greenfield effort?

Reply:
It's greenfield